templates/commands/render_action_regexreplace.go

```diff
@@ -116,7 +116,7 @@ func replaceWithTemplate(allMatches [][]int, b []byte, rr *model.RegexReplaceEnt
 			}
 		}
 		replaceAtStartIdx := oneMatch[subgroupNum*2] // bounds have already been checked in the caller
-		replaceAtEndIdx := oneMatch[subgroupNum+1]
+		replaceAtEndIdx := oneMatch[subgroupNum*2+1]
```

Comment:
Does https://pkg.go.dev/regexp#Regexp.SubexpIndex make any of this easier?

Reply:
We actually do call that above, at line 113 😀